### ISSUE TITLE
Strange image artifact

### ISSUE THREAD

**Issue by SqueezedByte** [koreader/koreader#8748]
KOReader version: 2022.01-57 
Device: Kobo Sage (same thing on Forma and Libra 2)
#### Issue
Strange artifact in picture.
#### Steps to reproduce
If picture has some gray level it appears some strange artifact on the picture
the picture with KOREADER is the image opened with Koreader
![Koreader](https://user-images.githubusercontent.com/33869313/151677233-2abec0a7-28ae-4f3a-9949-889cb1050b90.jpg)
The picture with kobo is the same file but opened with the standard reader of Kobo
![Lettore Kobo](https://user-images.githubusercontent.com/33869313/151677264-05449f94-049c-445c-b1f6-addfa6baf4cb.jpg)

I'm missing some setting ?
Thanks in advance for your help

**Comment by Frenzie** [comment]
You didn't say what kind of file it is. If it's an epub, I think we still don't default to "best" image scaling there although I think we should.

**Comment by SqueezedByte** [comment]
Sorry, I forgot about the file kind.
Is a .cbz.

The single picture is a .jpg
dimension : 1296x1920 pixel
resolution 96dpi
24bit

**Comment by Frenzie** [comment]
Strange, pinging @NiLuJe

**Comment by NiLuJe** [comment]
That's a color decimation artefact. There's no hardware dithering support on the Sage, nor do we do any kind of software dithering in PDFs.

TL;DR: Preprocess your CBZs properly ;).

(On mobile right now, but I have a script for that).

**Comment by NiLuJe** [comment]
Specifically, [this](https://svn.ak-team.com/svn/Bash/trunk/CBZ_gen.sh), which is simply [my usual set of image processing](https://www.mobileread.com/forums/showpost.php?p=3728291&postcount=17) tailored for the CBZ use-case.

I may look at enabling the dithering toggle w/ software dithering tomorrow...

**Comment by mergen3107** [comment]
Also have a look here: https://github.com/koreader/koreader/issues/2835#issuecomment-906768578
This is how I do processing for manga for reading on e-ink.
